cocos/input/types/event/event-mouse.ts

```diff
@@ -103,7 +103,7 @@ export class EventMouse extends Event {
      * @en The unique ID of SystemWindow, which triggerd the event
      * @zh 触发此事件的系统窗口 ID
      */
-    public windowId = 0;
+    public windowId = 1;
```

Comment:
Why change default value to 1?

Reply:
It should be 1 as native.

Reply:
0 is invalid id, but default value use it is ok as it should be assigned to a valid id.